src/create-extension-pack.js

```diff
@@ -45,6 +45,12 @@ const packName = 'builtin-extension-pack';
 const publisher = 'eclipse-theia';
 const repository = 'https://github.com/eclipse-theia/vscode-builtin-extensions';
 
+/**
+ * ms-vscode extensions known to be available in the 'Open VSX Registry'
+ * The source code of ms-vscode extensions is not included in the vscode repository but are however fetched by vscode at build time
+ */
+const msVscodeBuiltins = ['node-debug', 'node-debug2', 'references-view', 'js-debug-companion', 'js-debug'];
```

Comment:
@alvsan09 we might want to include a reference from where these extensions come from:

```suggestion
/**
 * The list of known `ms-vscode` builtin extensions available in the `open-vsx` registry, and which are included by default in vscode.
 * https://github.com/microsoft/vscode/blob/1.57.0/product.json#L34-L126
 */
const msVscodeBuiltins = ['node-debug', 'node-debug2', 'references-view', 'js-debug-companion', 'js-debug'];
```